test/components/DateRangePickerInputController_spec.jsx

```diff
@@ -113,33 +113,168 @@ describe('DateRangePickerInputController', () => {
   describe('#onEndDateChange', () => {
     describe('is a valid end date', () => {
       const validFutureDateString = moment(today).add(10, 'days').format('YYYY-MM-DD');
-      it('calls props.onDatesChange with correct arguments', () => {
-        const onDatesChangeStub = sinon.stub();
-        const wrapper =
-          shallow(<DateRangePickerInputController onDatesChange={onDatesChangeStub} />);
-        wrapper.instance().onEndDateChange(validFutureDateString);
-        expect(onDatesChangeStub.callCount).to.equal(1);
+      describe('when props.startDate is not provided', () => {
+        it('calls props.onDatesChange with provided end date', () => {
+          const onDatesChangeStub = sinon.stub();
+          const wrapper =
+            shallow(<DateRangePickerInputController onDatesChange={onDatesChangeStub} />);
```

Comment:
please don't add line breaks after assignments (same throughout, even if that was already in the file); if this is too long, let's indent like so:
```jsx
const wrapper = shallow((
  <DateRangePickerInputController onDatesChange={onDatesChangeStub} />
));
```